case-lib/lib.sh

```diff
@@ -802,6 +802,10 @@ set_alsa_settings()
 {
     # ZEPHYR platform shares same tplg, remove '_ZEPHYR' from platform name
     local PNAME="${1%_ZEPHYR}"
+
+    # Initialize alsa settings first
```

Comment:
Maybe this can be an echo rather than a comment?  Likewise with the other comment below.  The only `set -x` I saw in this file were wrapped in sub-shells. Approving regardless.

Reply:
I agree with @greg-intel , this looks like a big change and it seems very "quiet": I tried this command on my system and absolutely nothing was printed. Also, `dlogi "Run alsa setting for $PNAME"` runs AFTER this, could be confusing.

So either a `dlogi` or maybe just this:
```diff
-  alsactl init
+  alsactl --debug init
```

Reply:
> So either a new dlogi or maybe just this...

@fredoh9 , ping?

Reply:
I lost the momentum, need to review what was my intention at first